pkg/proxy/proxy.go

```diff
@@ -240,3 +260,7 @@ func (p *Proxy) roundTripperForRestConfig(config *rest.Config) (http.RoundTrippe
 func (p *Proxy) OIDCTokenAuthenticator() authenticator.Token {
 	return p.tokenAuther
 }
+
+func (p *Proxy) RunShutdownHooks() error {
```

Comment:
nit: `RunPreShutdownHooks`? for consistency

Reply:
:+1: